src/medVtkInria/vtkDataManagement/vtkMetaDataSetSequence.cxx

```diff
@@ -823,12 +823,11 @@ double vtkMetaDataSetSequence::GetMinTime() const
 //----------------------------------------------------------------------------
 double vtkMetaDataSetSequence::GetMaxTime() const
 {
-  double ret = 0;
-  for (unsigned int i=0; i<this->MetaDataSetList.size(); i++)
-    if ( ret < this->MetaDataSetList[i]->GetTime())
-      ret = this->MetaDataSetList[i]->GetTime();
-  return ret;
-  
+    double ret = 0;
```

Comment:
putting back this function, the indentation has been updated. There is no change in the code here